test/helper.rb

```diff
@@ -360,6 +360,14 @@ def md.directory?
       # oracle
       "bash -c 'type \"sqlplus\"'" => cmd.call('oracle-cmd'),
       "ef04e5199abee80e662cc0dd1dd3bf3e0aaae9b4498217d241db00b413820911" => cmd.call('oracle-result'),
+      # nginx mock cmd
+      %{nginx -V 2>&1} => cmd.call('nginx-v'),
+      %{/usr/sbin/nginx -V 2>&1} => cmd.call('nginx-v'),
+      %{bash -c 'type "/usr/sbin/nginx"'} => cmd.call('bash-c-type-nginx'),
+      # needed for two differnt inspec.command call formats
+      # host resource: dig commands
```

Comment:
Lines 368-370 are not related to this PR and are probably from a bad merge conflict resolution given the duration this PR has been open. Can you please remove them?